core/include/thread.h

```diff
@@ -11,6 +11,97 @@
  * @ingroup     core
  * @brief       Support for multi-threading
  *
+ * Priorities
+ * ==========
+ *
+ * As RIOT is using a fixed priority scheduling algorithm, threads are
```

Comment:
either `@ref core_sched "scheduling algorithm"` or

```markdown
[scheduling algorithm][core_sched]
```

I prefer the latter, but this sometimes does not work.